code/index.html

```diff
@@ -1,30 +1,211 @@
-    <!-- dont forget to link the javascript file here! -->
+
+
+    <!-- Accordion -->
+
+    <section class="middleSection">
+    <!-- <div class="left"> -->
+    <div class="accordion">
+      <h2 class="sectionHeading">Frequently Asked Questions</h2>
+      <div class="question" id="section1">
+        <h2>How can I find the time to meditate in my busy schedule?</h2>
+        <img class="expand-icon" id="icon" src="plus.svg" alt="plus icon">
+      </div>
+      <div class="answer" id="expanded">
+        <p>Well, the great thing about meditation is that it doesn’t really take all that much time. As soon as you pick up the basics, you can begin by practicing for five or ten minutes each day. Mornings are generally best, at least to start..</p>
+      </div>
+      <div class="question" id="section2">
+        <h2>How do I know if I’m meditating the right way?</h2>
+        <img class="expand-icon" id="icon" src="plus.svg" alt="plus icon">
+      </div>
+      <div class="answer" id="expanded">
+        <p>This question reflects the goal-oriented perfectionist in you who monitors your activities to make sure you’re doing them right. The great thing about meditation is that you can’t do it wrong, short of not doing it at all. (In fact, it’s the perfectionist that causes most of your stress. And the point of meditation is to reduce stress, not intensify it.)</p>
+      </div>
+      <div class="question" id="section3">
+        <h2>Can meditation really improve my health?</h2>
+        <img class="expand-icon" id="icon" src="plus.svg" alt="plus icon">
+      </div>
+      <div class="answer" id="expanded">
+        <p>Yes, meditation can make you healthier, both physically and psychologically! Researchers have published hundreds of studies investigating the health benefits of meditation, and the results consistently indicate that people who meditate regularly have better health than those who don’t. Besides, meditation can actually reduce anxiety and lift depression.</p>
+      </div>
+      <div class="question" id="section4">
+        <h2>What should I do if I keep falling asleep while I meditate?</h2>
+        <img class="expand-icon" id="icon" src="plus.svg" alt="plus icon">
+      </div>
+      <div class="answer" id="expanded">
+        <p>Like restlessness, sleepiness is a common roadblock on the journey of meditation. First, you may want to explore the sleepiness a little. Where do you experience it in your body? Is it just mental dullness, or are you physically tired as well? Perhaps you should be napping rather than meditating!</p>
+      </div>
+    </div>
+
+    <div class="quotes1">
+       <div class="natthiko">
+          <span class="quote">"Be kind to yourself and your practice. Benefits can take time and patience."</span>
+         <span class="writer">/Headspace</span>
+       </div>
+        <div class="natthiko">
+         <span class="quote">"Don´t believe everything you think. They are only thoughts."</span>
```

Comment:
A little out of context, but I really liked these quotes :)

Reply:
😍